### PR TITLE
Update contributing documentation for 1.3 branch

### PR DESCRIPTION
**Pull request by jmleroux**
Update the contribution instructions to the last version

### CODE REVIEW

**Comment by grena** [comment]
GTM

**Comment by solivier** [comment]
GTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,25 +1,72 @@
...
+```
+
+Extra info
+----------
+
+Specs means phpspec have been written, every classes are specced except controllers, form types, commands, doctrine entity (POPO), symfony semantic config.
+
+Behats means behat scenario have been written, for nominal and limit cases, internal api can be also tested through behat via commands (like query or updater).
+
+Blue CI means,
```
Comment (filipsalpe) [inline CONTRIBUTING.md]: Why not green?

Comment (jjanvier) [inline CONTRIBUTING.md]: In Jenkins, it's blue by default ;)

Comment (filipsalpe) [inline CONTRIBUTING.md]: https://wiki.jenkins-ci.org/display/JENKINS/Green+Balls :green_heart:

**Pull request merged**